meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt

```diff
@@ -1,20 +1,22 @@
 package com.gdgistanbul.api
 
+import com.gdgistanbul.model.Attendance
 import com.gdgistanbul.model.Event
-import retrofit2.http.Field
-import retrofit2.http.GET
-import retrofit2.http.Path
-import retrofit2.http.Query
+import retrofit2.http.*
 
 interface MeetupApi {
 
-    @GET("events")
-    suspend fun getEvents(@Query("has_ended") hasEnded: Boolean = true): List<Event>
+    @GET("{url_name}/events")
+    suspend fun getEvents(@Path("url_name") group: String = "GDGIstanbul"): List<Event>
 
-    @GET("events/{id}/attendance")
-    suspend fun getAttendees(@Path("id") id: String, @Query("order") order: String = "name")
+    @GET("{url_name}/events/{id}/attendance")
+    suspend fun getAttendees(
+        @Path("group") group: String = "GDGIstanbul",
+        @Path("id") id: String,
+        @Query("order") order: String = "name"
+    ): List<Attendance>
 
 
-    @GET("events/{id}/attendance")
-    suspend fun getAttendenceTaking(@Field("member") id: String, @Field("status") status: String )
+    @POST("{url_name}/events/{id}/attendance")
```

Comment:
add url_name path parameter to the parameters

Reply:
fixed